components/sidebar.jsx

```diff
@@ -71,7 +80,7 @@ var Sidebar = React.createClass({
           <Link to="home">
             <img src="/img/nav/mozilla-wordmark-white.svg" alt="Webmaker logo" className="moz-logo"/>
           </Link>
-          <span aria-label="toggle" role="button" onKeyUp={this.handleHamburgerClick} onKeyDown={this.handleHamburgerClick} className="glyphicon glyphicon-menu-hamburger hidden-lg hidden-md"
+          <span aria-label="toggle" role="button" onKeyDown={this.handleHamburgerKeyDown} className="glyphicon glyphicon-menu-hamburger hidden-lg hidden-md"
```

Comment:
Was this the only issue you had with the PR? I wasn't sure, so I re-marked this one as "PR needs review"...